lerna.json

```diff
@@ -1,6 +1,18 @@
 {
+  "version": "5.0.0",
   "npmClient": "yarn",
-  "version": "independent",
+  "npmClientArgs": ["--no-lockfile"],
```

Comment:
Is this desired? Why adding now?

Reply:
I noticed this in the babel `lerna.json` since we are .gitignoring these already this is equivalent.